agents/a2a_mcp_integration.py

```
@@ -297,345 +324,494 @@ async def _send_mcp_pipe(self, message: A2AMCPMessage) -> Dict[str, Any]:
         self.running = True
         while self.running:
             await self._update_stats()
             await asyncio.sleep(5)  # Update every 5 seconds
-    
+
     def stop(self):
         """Stop performance monitoring"""
         self.running = False
-    
+
     async def _update_stats(self):
```

Ruff (0.11.9)

Missing return type annotation for private function `_update_stats`

Add return type annotation: `None`

(ANN202)